project/AppDependencies.scala

```diff
@@ -1,19 +1,16 @@
-import play.core.PlayVersion
-import play.sbt.PlayImport._
-import sbt.Keys.libraryDependencies
-import sbt._
+import sbt.*
 
 object AppDependencies {
 
-  private val bootstrapVersion = "9.0.0"
+  private val bootstrapVersion = "9.1.0"
```

Comment:
Should this not be 9.2.0 going off the catalogue?

Reply:
My eyes have truly deceit me. I was checking the catalogue and I could swear I saw 9.1.0 , thanks for noticing

Reply:
:laughing: it might have just updated

Reply:
Either that or I need a jug of expresso